src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,40 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represent a parent class for a Person's contacts
+ */
+public class Contact {
+	private String value;
+	private boolean isPrivate;
+	
+	public Contact() {}
```

Comment:
An empty constructor like this adds no value(it is there by default in Java). Think twice if you have to leave empty constructor like this in your code. There must be something more you need to do. 

A super class will most of the time share a non-empty common constructor with its child classes if the chiild classes are accepting arguments in their constructors.